app/controllers/logins_controller.rb

```diff
@@ -0,0 +1,15 @@
+class LoginsController < ApplicationController
+
+  def login_admin
+  end
+
+  def index
+
+  end
+
+  def menu
+
+  end
+
+
```

Comment:
refactor this controller setting they method in one line like this -> `def login_admin; end`